toolchain/check/member_access.h

```diff
@@ -23,6 +23,10 @@ auto PerformCompoundMemberAccess(Context& context, Parse::NodeId node_id,
                                  SemIR::InstId base_id,
                                  SemIR::InstId member_expr_id) -> SemIR::InstId;
 
+auto PerformTupleIndex(Context& context, Parse::NodeId node_id,
```

Comment:
We try to keep comments on functions. Here's a suggestion similar to other functions in this file:

```suggestion
// Creates SemIR to perform a tuple index with base expression `tuple_inst_id` and
// index expression `index_inst_id`. Returns the result of the access.
auto PerformTupleIndex(Context& context, Parse::NodeId node_id,
```